tests/Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerContentBothDialog.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.AI.QnA;
+using Microsoft.Bot.Builder.AI.QnA.Dialogs;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.AI.Tests
+{
+    /// <summary>
+    /// QnAMaker action builder class.
+    /// </summary>
+    public class QnAMakerContentBothDialog : QnAMakerDialog
```

Comment:
I am not sure, why you need to these derived dialog classed in the tests folder.

didn't find similar example in existing one.

for example: recently 'RankerType' new param added in request, that test for that one is different  pattern (cross check)